go/src/koding/klientctl/endpoint/machine/info.go

```diff
@@ -0,0 +1,124 @@
+package machine
+
+import (
+	"fmt"
+	"math"
+	"time"
+
+	"koding/klient/machine"
+)
+
+// Info stores the basic information about the machine.
+type Info struct {
+	// ID is an unique identifier for a given machine.
+	ID string `json:"id"`
+
+	// Alias stores a human readable alias for machine.
+	Alias string `json:"alias"`
+
+	// Team is the name of the team which created the machine.
+	Team string `json:"team"`
+
+	// Stack describes machine parent stack.
+	Stack string `json:"stack"`
+
+	// Provider represents machine provider.
+	Provider string `json:"provider"`
+
+	// Label is the machine label, as seen by the Koding UI.
+	Label string `json:"label"`
+
+	// The IP of the running machine.
+	IP string `json:"ip"`
```

Comment:
I added `RegisterURL` and `QueryString` fields. However, I'm not going to show them in `kd machine list` due to their width and the fact that output from list command already has quite a few columns... 

They will be available in `kd machine list --json`